src/style.css

```diff
@@ -0,0 +1,152 @@
+@import url(./card.css);
+@import url(./star.css);
+
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto-Regular.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Roboton;
+  src: url(fonts/Roboto-Medium.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+@font-face {
+  font-family: Robotos;
+  src: url(fonts/Roboto-Bold.ttf);
+  font-style: normal;
+  font-weight: 700;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+h1 p {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.card {
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.imacImages {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 30.5px;
+}
+
+.title {
+  font-family: Roboton, sans-serif;
```

Comment:
the font is better to apply to the whole page

Reply:
yes it is better, but i can't add fonts for the whole page because i have different font for each title